ClientRuntimes/NodeJS/package.json

```diff
@@ -25,7 +25,8 @@
     "request": "2.45.0",
     "validator": "~3.1.0",
     "duplexer": "~0.1.1",
-    "through": "~2.3.4"
+    "through": "~2.3.4",
+    "tough-cookie": "*"
```

Comment:
jar - If true and **tough-cookie is installed**, remember cookies for future use (or define your custom cookie jar; see examples section) as per request.js doc